UI/data/locale/en-US.ini

```diff
@@ -55,6 +55,15 @@ Enable="Enable"
 DisableOSXVSync="Disable macOS V-Sync"
 ResetOSXVSyncOnExit="Reset macOS V-Sync on Exit"
 HighResourceUsage="Encoding overloaded! Consider turning down video settings or using a faster encoding preset."
+DisableOSXVSync="Disable OSX V-Sync"
+ResetOSXVSyncOnExit="Reset OSX V-Sync on Exit"
+LearnMore="Learn more"
+HighResourceUsage.Title="High Resource Usage"
+HighResourceUsage="Encoding overloaded!  Consider turning down video settings or using a faster encoding preset."
```

Comment:
Use a single space between sentences.

Reply:
When rebasing, old strings got into the commit. It is fixed now.